### DISCUSSION TITLE
bevel in plate editor

### DISCUSSION THREAD

**Discussion by nic-is** [idea-statica/ideastatica-public#291] [Help Q&A]
Can we bevel two sides of plate at the same time? if so, how to do it? i tried to enter 2 3 for Selected corners but a warning says 'Input value'2 3 not in a correct format'

**Comment by nathanlukeIDEA** [comment]
Hi @nic-is, I don't believe this is possible. You can get in touch with https://www.ideastatica.com/support-center for questions relating to the application.
